pom.xml

```diff
@@ -54,22 +55,27 @@
         <dependency>
             <groupId>org.jenkins-ci.test</groupId>
             <artifactId>docker-fixtures</artifactId>
-            <version>1.8</version>
+            <version>1.11</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>ssh-slaves</artifactId>
-            <version>1.30.0</version>
+            <version>1.31.5</version> <!-- TODO: Remove version when core is 2.277 or greater (included in BOM) -->
             <scope>test</scope>
         </dependency>
     </dependencies>
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>io.jenkins.tools.bom</groupId>
+                <artifactId>bom-2.249.x</artifactId>
+                <version>876.vc43b4c6423b6</version>
+            </dependency>
             <dependency>
                 <groupId>org.jenkins-ci</groupId>
                 <artifactId>symbol-annotation</artifactId>
-                <version>1.17</version>
+                <version>1.20</version>
             </dependency>
```

Comment:
I think you should be able to drop this `dependency` from `dependencyManagement` now that the BOM is being used.

Reply:
Yes, delete the whole `<dependency>`.

Reply:
done in 393fcc5eb12afdb0f08dd39e4813c4433a254678